src/Xamarin.Android.NUnitLite/Gui/Instrumentations/TestSuiteInstrumentation.cs

```diff
@@ -146,12 +146,22 @@ protected void AddTest (Assembly assembly)
 
 		protected virtual IEnumerable <string> GetIncludedCategories ()
 		{
-			return null;
+			string include = arguments?.GetString ("include");
+			if (!string.IsNullOrEmpty (include)) {
+				foreach (var category in include.Split (',')) {
+					yield return category;
+				}
+			}
 		}
 
 		protected virtual IEnumerable <string> GetExcludedCategories ()
 		{
-			return null;
```

Comment:
Instead of returning null, these will be the equivalent of `yield break`

Reply:
`TestSuiteActivity` should also be updated to not return `null`.